doc/rexml/context.rdoc

```diff
@@ -0,0 +1,140 @@
+== Element Context
+
+Notes:
+- All code on this page presupposes that the following has been executed:
+
+    require 'rexml/document'
+    include REXML
```

Comment:
I think that this may confuse readers.
How about using `REXML::` explicitly like `REXML::Document`?

Reply:
Done.